bftengine/include/bftengine/ControlStateManager.hpp

```diff
@@ -36,13 +36,17 @@ class ControlStateManager {
   bool getPruningProcessStatus() const { return onPruningProcess_; }
   bool getRestartBftFlag() const { return restartBftEnabled_; }
   void setRestartBftFlag(bool bft) { restartBftEnabled_ = bft; }
-
   void setRemoveMetadataFunc(std::function<void(bool)> fn) { removeMetadataCbRegistry_.add(fn); }
   void setRestartReadyFunc(std::function<void()> fn) { sendRestartReady_ = fn; }
   void sendRestartReadyToAllReplica() { sendRestartReady_(); }
+  void setInstallReadyFunc(std::function<void(const std::string&)> fn) { sendInstallReady_ = fn; }
```

Comment:
Can we somehow leverage the existing restart proof mechanism (and maybe change its name) instead of introducing another mechanism?

Reply:
fixed